include/circt/Dialect/HW/HWStructure.td

```diff
@@ -231,6 +231,10 @@ def HWModuleOp : HWModuleOpBase<"module",
 
     /// Verifies the body of the function.
     LogicalResult verifyBody();
+
+    void appendPorts(
+    ArrayRef<std::tuple<StringAttr, Type, Location>> insertInputs,
```

Comment:
nit: Can we tweak the formatting to not have these parameters on same column as the beginning of the method declaration? Thanks!

Reply:
Thanks for the suggestion, fixed.